src/section-variables-and-evaluating-expressions.ptx

```diff
@@ -105,8 +105,8 @@
     </subsection>
 
     <subsection>
-        <title>Mathematical Expressions</title>
```

Comment:
If you'd like to change "mathematical expression" to "algebraic expression", could we do that globally to every instance, and put it into its own narrow commit? Search with regex for `(?<=\W)([Aa] )?[Mm]athematical [Ee]xpression`. My search comes up with one that you missed.